--- config.def.h	2017-05-25 17:04:18.160071857 +0200
+++ config.h	2017-05-25 17:39:10.380188537 +0200
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-char font[] = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+char font[] = "-xos4-terminus-medium-*-*-*-18-*-*-*-*-*-*-*";
 int borderpx = 2;
 
 /*
@@ -89,7 +89,7 @@
 	"red3",
 	"green3",
 	"yellow3",
-	"blue2",
+	"#1930ea",
 	"magenta3",
 	"cyan3",
 	"gray90",
